.gitignore

```diff
@@ -60,4 +60,10 @@ xcuserdata
 Tests/Pods
 **/.toDelete
 /test-gh-pages-current/
-doc/Programming-guide/Server-REST-APIs/swagger.json
\ No newline at end of file
+doc/Programming-guide/Server-REST-APIs/swagger.json
+/server/containers/docker/using-compose/kaa-nginx-config/kaa-nginx.conf
+/server/containers/docker/using-compose/kaa-nginx-config/kaa-default.conf
+/logs
+/server/containers/docker/using-compose/kaa-docker-compose.yml
+/server/containers/docker/kaa-node.deb
```

Comment:
kaa-docker-compose.yml will be creating when you start launch-kaa.sh(script which configure kaa-node and start docker)
if you want download kaa-node.deb from http://www.kaaproject.org/download-kaa/
you can locate this file in /server/containers/docker/ folder and run sh build.sh for creating docker image